### PR TITLE
Added bor fingerprint

### PR DESCRIPTION
**Pull request by 0xsharma**
Added bor fingerprint command to the new CLI. It works offline and shows all the properties listed in https://linear.app/matic/issue/POS-55/bor-fingerprint.

### CODE REVIEW

File: internal/cli/bor_fingerprint.go [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+	cores := 0
+	for i := 0; i < len(cp); i++ {
+		cores += int(cp[i].Cores)
+	}
+	return cores
+}
+
+// Run implements the cli.Command interface
+func (c *FingerprintCommand) Run(args []string) int {
+	v, _ := mem.VirtualMemory()
```
Comment (ferranbt) [inline internal/cli/bor_fingerprint.go:42]: Do never skip an error.

File: internal/cli/bor_fingerprint.go [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+	h, _ := host.Info()
+	cp, _ := cpu.Info()
+	d, _ := disk.Usage("/")
+
+	osName := h.OS
+	osVer := h.Platform + " - " + h.PlatformVersion + " - " + h.KernelArch
+	totalMem := math.Floor(float64(v.Total)/(1024*1024*1024)*100) / 100
```
Comment (ferranbt) [inline internal/cli/bor_fingerprint.go:49]: Please create a function for this.

File: internal/cli/bor_fingerprint.go [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+	availableMem := math.Floor(float64(v.Available)/(1024*1024*1024)*100) / 100
+	usedMem := math.Floor(float64(v.Used)/(1024*1024*1024)*100) / 100
+	totalDisk := math.Floor(float64(d.Total)/(1024*1024*1024)*100) / 100
+	availableDisk := math.Floor(float64(d.Free)/(1024*1024*1024)*100) / 100
+	usedDisk := math.Floor(float64(d.Used)/(1024*1024*1024)*100) / 100
+
+	borDetails := fmt.Sprintf("Bor Version : %s", params.VersionWithMeta)
+	cpuDetails := fmt.Sprintf("CPU : %d cores", getCoresCount(cp))
+	osDetails := fmt.Sprintf("OS : %s %s ", osName, osVer)
```
Comment (ferranbt) [inline internal/cli/bor_fingerprint.go:58]: See other commands, for structured info like this, we are not using Output directly but a prettify version of it using `formatKV` and `formatList`.

**Review by ferranbt — CHANGES_REQUESTED** [review]
Some initial review.

**Review by ssandeep — DISMISSED** [review]

**Review by manav2401 — DISMISSED** [review]

**Review by vcastellm — APPROVED** [review]
GLTM but please request against `v0.3.0-dev`

**Comment by Shivam691999** [comment]
> GLTM but please request against `v0.3.0-dev`

Done @vcastellm

**Review by vcastellm — APPROVED** [review]

**Review by vcastellm — CHANGES_REQUESTED** [review]

File: internal/cli/bor_fingerprint.go [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+}
+
+// Checks if fio exists on the node
+func (c *FingerprintCommand) checkFio() error {
+
+	cmd := exec.Command("/bin/sh", "-c", "fio -v")
+
+	_, err := cmd.CombinedOutput()
+	if err != nil {
+		message := "\nFio package not installed. Install Fio for IOPS Benchmarking :\n\nDebianOS  :  'sudo apt-get update && sudo apt-get install fio -y'\nAWS AMI/CentOS  :  'sudo yum install fio -y'\nOracle LinuxOS  :  'sudo dnf install fio -y'\n"
```
[the remark this answers is not in the document]
Comment (Shivam691999) [reply internal/cli/bor_fingerprint.go:83]: Yes @vcastellm , it already does that!

**Review by temaniarpit27 — APPROVED** [review]

**Review by vcastellm — APPROVED** [review]
LGTM

**Review by ferranbt — APPROVED** [review]

**Pull request merged**